CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This file is used to list changes made in each version of the yum-remi-chef  coo
 - Various Cookstyle fixes
 - Add InSpec tests
 - Add php 7.4 support
+- Convert inspec tests to inspec profiles
```

Comment:
This needs to be moved up to the `Unreleased` section.

Reply:
oops